queries/unison/indents.scm

```diff
@@ -0,0 +1,19 @@
+[
+  (kw_then)
+  (cases)
+  (kw_equals)
+  (kw_else)
+  (do)
+] @indent.begin
+
+(literal_list) @indent.begin
+
+(tuple_or_parenthesized
+  "(") @indent.begin
```

Comment:
Shouldn't there also be a 
```
(tuple_or_parenthesized
  "}" @indent.branch @indent.end)
```
to go with this

Reply:
There should! However this made an indent action (e.g. pressing `o`) slower, there was a noticeable delay when that query is used as well as:
```scheme
(literal_list
  ")" @indent.branch @indent.end)
```

So I opted to include only the starting queries

Reply:
Yes, but won't that break indentation (in some situations)? Correctness > performance

If the performance is unacceptable, it's better not to touch the construct at all.

Reply:
In the (limited) tests I did I didn't see weird behaviour.

However you are right, I'd rather see it being correct. I've removed the queries, because the performance hit was disturbing, and adding them back in shouldn't be that hard.